localslackirc/irc.py

```diff
@@ -991,13 +990,9 @@ async def irc_listener() -> None:
                 to_irc_task,
             )
         finally:
-            log('Closing connections')
-            sl_client.close()
-            if status_file:
-                log(f'Writing status to {status_file}')
-                status_file.write_bytes(sl_client.get_status())
```

Comment:
Case in point: this commit claims to touch logging, yet it removes saving the status file.

Big surprise that I didn't accept your changes…

Reply:
I suggest you to have a look at the end of daemon.py.

Reply:
But you're right there was a bug, shutdown wasn't correctly handled, and the status file wasn't written.

Fix: https://github.com/rbignon/localslackirc/commit/8bb410ab6c819708e975e7fd16ebdc563828ba6c